### PR TITLE
ENSWEB-4340: make gene tree in e!g look similar to e! gene tree

### PR DESCRIPTION
**Pull request by imran-ebi**
These are some additional changes to ENSWEB-4340 JIRA ticket. The code changes should make e!g gene tree appear similar to e! gene tree. Also fixed other within-species paralogues being highlighted when highlighting is disabled.

### CODE REVIEW

**Comment by ens-ma7** [comment]
HI @imran-ebi can you provide sandbox URL please.

File: modules/EnsEMBL/Draw/GlyphSet/genetree.pm [hunk 1]
```diff
@@ -188,23 +189,26 @@ sub _init {
...
-      } elsif( $f->{_genes}->{$current_gene} ){
-        $label_colour     = 'red';
-        $collapsed_colour = 'red';
-        $node_colour = 'navyblue';
-        $bold = defined($other_genome_db_id);
-      } elsif( $f->{_genome_dbs}->{$current_genome_db_id} ){
-        $label_colour     = 'blue';
-        $collapsed_colour = 'navyblue';
-        $bold = defined($other_genome_db_id);
+    if ( $f->{label} !~ m/homologs/ ) {
```
Comment (ens-ma7) [inline modules/EnsEMBL/Draw/GlyphSet/genetree.pm]: Is it not possible to merge both if like this: if($f->{label} && $f->{label} !~ m/homologs/)

Comment (imran-ebi) [reply modules/EnsEMBL/Draw/GlyphSet/genetree.pm]: I have tried that before with Perl, but it does not work unlike in JavaScript for some reason. That is why I separated the two conditions. I'll try to merge them both and see if it works with this code though.

File: modules/EnsEMBL/Draw/GlyphSet/genetree.pm [hunk 2]
```diff
@@ -46,9 +46,10 @@ sub _init {
 ## EG
   my $highlight_annotations   = $self->{highlights}->[8]; # EG
 ##
-  my $tree          = $self->{'container'};
-  my $Config        = $self->{'config'};
-  my $bitmap_width = $Config->image_width(); 
+  my $tree           = $self->{'container'};
+  my $Config         = $self->{'config'};
+  my $highlight_gene = $Config->{_parameters}->{highlight_gene};
```
Comment (ens-ma7) [inline modules/EnsEMBL/Draw/GlyphSet/genetree.pm]: Can you use this instead to get the parameter:
$Config->get_parameter('highlight_gene'); 
Let me know if it works

Comment (imran-ebi) [reply modules/EnsEMBL/Draw/GlyphSet/genetree.pm]: It worked. Thanks 😄

**Comment by imran-ebi** [comment]
@ens-ma7 Sorry I couldn't add it to the description. The link is: http://ves-hx-78.ebi.ac.uk:9200/Chondrus_crispus/Gene/Compara_Tree/pan_compara?anc=12149838;collapse=12149968,12150242,12150153;db=core;g=CHC_1040;g1=Npun_R3807;r=Pt:155059-155592;t=CCP38213.

**Pull request merged**